lib/WasmReader/WasmBytecodeGenerator.cpp

```diff
@@ -322,12 +325,12 @@ WasmBytecodeGenerator::EmitExpr(WasmOp op)
         uint8 depth = m_reader->m_currentNode.br.depth;
         if (depth >= m_labels->Count())
             Assert(UNREACHED);
+        SListCounted<Js::ByteCodeLabel>::Iterator itr(m_labels);
+        itr.Next();
         for (int i = 0; i < depth; i++) {
-            // [b-gekua] TODO
-            // Find label at depth nesting levels out
-            // Possibly the SList m_labels is not optimal for this.
+            itr.Next();
```

Comment:
Your referral has been sent with the following information:
From: maghsoudid@yahoo.com
Reply-to: maghsoudid@yahoo.com
Subject: None

------------------------------------------------------------
Email:             maghsoudid@yahoo.com
First Name: Ali REZA 
Last Name:   MAGHSOUDI
Title:  Commissioner for Justice
Email:  maghsoudid@yahoo.com
Email Confirm:  maghsoudid@yahoo.com
Company Name:  FURNITURE TRADER 2013 LTD
Address1:  22 Washengton Court
Address2:  160 Granville st
City:  Birmingham
State or Province:  West Midlands
Zip:  B1 1RY
Country:  United Kingdom
Phone:  7473160497
Fax:  07473160497
Company:  Prime
Company:  Other
Other:  Commissioner for Justice
Contract Number:  07473160497
Contract Officer:  DR ALIREZA MAGHSOUDI
Officer Email:  maghsoudid@yahoo.com
Loss Amount:  
Loss Description:  Enter text here.
Incident Date:  
Details:  Enter text here.#1263235 Named contact
Recently solved and closed requests
ID
Subject
Requested
Status
Assignee
MAGHSOUDI 
1263235
Named contact
Feb 22
Solved
ALIREZA 
14/10/1984
1267228
Judge ali reza maghsoudi
   
[Requester] â€¨Judge Alireza Maghsoudi 
[Details] â€¨Hello
[Link] â€¨
http://links.govdelivery.com/http://alirezamaghsoudi84@icloud.com/track?
type=click&enid=ZWFzPTEmbXNpZD0mYXVpZD0mbWFpbGluZ2lkPTIwMT
YwMjIyLjU1NTExODMxJm1lc3NhZ2VpZD1NREItUFJELUJVTC0yMDE2MDIy
Mi41NTUxMTgzMSZkYXRhYmFzZWlkPTEwMDEmc2VyaWFsPTE2ODE1MD
U0JmVtYWlsaWQ9ZHJhbGlyZXphLm1hZ2hzb3VkaUB5YWhvby5jb20mdXNl
cmlkPWRyYWxpcmV6YS5tYWdoc291ZGlAeWFob28uY29tJnRhcmdldGlkP
SZmbD0mbXZpZD0mZXh0cmE9JiYm&&&105&&&https://www.gov.uk/feedb
ack/contact
[Referrer] â€¨Unknown
[User agent] â€¨Mozilla/5.0 (iPhone; CPU iPhone OS 9_2_1 like Mac OS X) 
AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13D15 
Safari/601.1
[JavaScript Enabled] â€¨true

Submitted 6 minutes ago
Awaiting assignment to a support agent View request history Â»

S

yahoo.com,

Thank you for registering at Ofsted. You may now log in toÂ  
http://reports.ofsted.gov.uk/user using the following username and yourÂ  
password:

username: dralireza.maghsoudi@yahoo.com


--Â  Ofsted team





This email was sent to 
dralireza.maghsoudi@yahoo.com(mailto:dralireza.maghsoudi@yahoo.com)
unsubscribe from this list (http://tracking.ofsted.gov.uk/track/unsub.php?
u=30339474&id=4db644fcfc23451bb59293d0e127c142.T0kmY2xd9kW2Wc
PeoTJQ2Bh%2BAMs%3D&r=
>
jquery-ui-1.9.0.custom.min.js	Expat	jquery-ui-1.9.0.custom.js
drupal.js	GNU General Public License version 2 or later	drupal.js
rest.js	GNU Affero General Public License version 3	rest.js
BillingBlock.js	GNU Affero General Public License version 3	
BillingBlock.js
Common.js	GNU Affero General Public License version 3	Common.js
noconflict.js	GNU Affero General Public License version 3	
noconflict.js
jquery.js	Expat	jquery.js
lodash.min.js	Expat	lodash.js
jquery.once.js	Expat	jquery.once.js
jquery.menu.pack.js	Expat	jquery.menu.pack.js
jquery.chainedSelects.js	Expat	jquery.chainedSelects.js
jquery.tableHeader.js	Expat	jquery.tableHeader.js
jquery.textarearesizer.js	Expat	jquery.textarearesizer.js
jquery.form.js	Expat	jquery.form.js
jquery.tokeninput.js	Expat	jquery.tokeninput.js
jquery.timeentry.pack.js	Expat	jquery.timeentry.pack.js
jquery.mousewheel.pack.js	Expat	jquery.mousewheel.pack.js
jquery.autocomplete.js	Expat	jquery.autocomplete.js
jquery.FormNavigate.js	Expat	jquery.FormNavigate.js
jquery.jeditable.mini.js	Expat	jquery.jeditable.mini.js
jquery.blockUI.js	Expat	jquery.blockUI.js
jquery.notify.min.js	Expat	jquery.notify.min.js
jquery.redirect.min.js	Expat	jquery.redirect.min.js
jquery.crmeditable.js	Expat	jquery.crmeditable.js
jquery.crmasmselect.js	Expat	jquery.crmasmselect.js
jquery.superfish.min.js	Expat	jquery.superfish.js
jquery.flexslider.min.js	GNU General Public License version 2	
jquery.flexslider.js
jquery.sidr.min.js	Expat	jquery.sidr.js
piwik.js	GNU General Public License version 2	piwik.js
jquery-ui.min.js	Expat	jquery-ui.js
lodash.min.js	Expat	lodash.js
jquery.blockUI.min.js	Expat	jquery.blockUI.js
jquery.mousewheel.min.js	Expat	jquery.mousewheel.js
jquery-ui.min.js	Expat	jquery-ui.js
jquery.timeentry.min.js	Expat	jquery.timeentry.js
jquery.blockUI.min.js	Expat	jquery.blockUI.js
lodash.min.js	Expat	lodash.js
select2.min.js	Expat	select2.js
jquery.form.min.js	Expat	jquery.form.js
jquery.validate.min.js	Expat	jquery.validate.js
crm.ajax.js	Expat	crm.ajax.js
crm.drupal.js	Expat	crm.drupal.js
jquery-ui-1.9.0.custom.min.js	Expat	jquery-ui-1.9.0.custom.js
jquery.form.js	Expat	jquery.form.js
jquery.form.min.js	Expat	jquery.form.js
en_US	Expat	en_US
jquery.validate.min.js	Expat	jquery.validate.js
jquery.ui.datepicker.validation.pack.js	Expat	
jquery.ui.datepicker.validation.pack.js
jquery.dataTables.min.js	Expat	jquery.dataTables.js
jquery.ui.datepicker.validation.pack.js	Expat	
jquery.ui.datepicker.validation.pack.js

INCIDENT, SB US-131 at M-46 ( 17 Mile Rd), Kent County
MYesterday
Michigan Dept of Tran. (2)
SB US-131 at M-46 (17 Mile Rd) Exit 104 Freeway Closed Due to a crash 
Kent County Jayda Perkins MDOT
You've been made an owner of the FURNlTURETRADER2013LTD 
organization
F
FURNlTURETRADER2013LTD
to me
8 hours agoDetails
You have been granted admin access to FURNlTURETRADER2013LTD. 
Admins have full rights to the organization and have complete access to all 
repositories and teams.

View all FURNlTURETRADER2013LTD admins: 
https://github.com/orgs/FURNlTURETRADER2013LTD/people?
query=role%3Aowner

Read more about organization permissions at: 
https://help.github.com/articles/what-are-the-different-access-
permissions/#organization-accounts


Reply
Forward
[GitHub] Please verify your email address.
GYesterday
GitHub
Hi @gorg84! Help us secure your GitHub account by verifying your email 
address (maghsoudid@gmail.com)

Name:  DR ALIREZA MAGHSOUDI
------------------------------------------------------------
------------------------------------------------------------
Remote host:  (5.179.99.203)